Updating Docs home (#78)
spokestack/spokestack-website: content/docs/welcome.md

---

> title: Spokestack Documentation

Reviewer: ```suggestion
title: Welcome
```

> navId: Spokestack Documentation 

Reviewer 2: If you change the `navId`, you have to change the reference to the old one in the `nav.json` file too.

Reviewer: The `title` and `navId` should probably stay unchanged. All pages are "Spokestack Documentation".

> navId: Spokestack Documentation 

Reviewer: ```suggestion
navId: Welcome
```

description: Spokestack documentation for mobile and web voice interface development
draft: false
---

**Experienced voice _and_ mobile developer? Start here...**

If you're looking to integrate Spokestack in your project as fast as possible, our quickstart guides for each platform can be found here:

- [iOS](/docs/iOS/getting-started)
- [Android](/docs/Android/getting-started)
- [React Native](/docs/React Native/getting-started)

**Would you rather start with a tutorial?**

> - [Swift](/blog/create-an-alexa-compatible-dialog-manager-in-swift)

Reviewer 2: I wouldn't link directly to the dialog manager tutorial; I'd just link to the first part of the series instead of breaking it down by language here. That first post has a good table of contents that people can use to find what they want.

> - [Swift](/blog/create-an-alexa-compatible-dialog-manager-in-swift)

Reviewer: ```suggestion
- [Swift](/blog/porting-a-smart-speaker-voice-app-to-mobile-part-1)
```

- [Java] Coming soon!

> If you do not find what you are looking for in our documentation, leave a note in our [forum](https://forum.spokestack.io/) and we'll address it there.

Reviewer 2: ```suggestion
If you have any other questions, please check our [forum](https://forum.spokestack.io/), which we monitor closely. You might find an answer there already.
```
I'd put this at the very end, below the next section. It already lists a few ways to address missing documentation.
